pg_dba_01/chapters/PostgreSQL_glance.tex

```diff
@@ -124,28 +123,28 @@ \subsection{Multi platform support}
 
 \subsection{Tablespaces}
 The tablespace support permits a fine grained distribution of the data files across
-filesystems. In \ref{sub:TBS-LOGICAL} and \ref{sub:TBS-PHYSICAL} we'll see how to take advantage of 
+filesystems. In chapters \ref{sub:TBS-LOGICAL} and \ref{sub:TBS-PHYSICAL} we'll see how to take advantage of 
 this powerful feature.
 
 \subsection{Triggers} 
-The triggers are well supported on tables and views. A basic implementation of the events 
-triggers is also present. The triggers can emulate completely the updatable views feature. 
+The triggers are well supported on tables and views. A basic implementation of event 
+triggers is also present. The triggers can completely emulate the updatable views feature. 
 
 \subsection{Views}
 The read only views are well consodlidated in PostgreSQL.
-The version 9.3 introduced the basic support for the materialised and updatable views.
+Version 9.3 introduced basic support for the materialised and updatable views.
 For the materialised views there is no incremental refresh. The complex views, like views 
-joining two or more tables, are not updatable. 
+joining two or more tables, cannot be updated. 
```

Comment:
I would leave the previous statemen because the updatable views is a feature which is disabled under certain conditions . however the tables are updatable and a complex view can still become updatable using the triggers.